.gitignore

```diff
@@ -0,0 +1,3 @@
+.DS_Store
+jordan.xcuserdatad
+/JordanResources
```

Comment:
I went ahead and added a global gitignore so you can wipe yours and do a pull from develop and we golden 👌